docs/whatsnew_1.5.7.html

```diff
@@ -173,19 +173,28 @@ <h2>
                 The most notable improvements and bugfixes in v1.5.7 since v1.5.6 include:
             </h2>
             <ul>
-                <li>Core - Updated hundreds of files for compatibility with PHP 7.2, 7.3 and 7.4</li>
+                <li>Core - Updated hundreds of files for compatibility with PHP 7.2, 7.3 and 7.4.  <b>These changes, along with updating your PHP version, will make your cart faster.</b></li>
                 <li>Admin - Default notification behavior for order update (send email/do not send email/hide update) may now be set</li>
                 <li>Admin - Admins can now login to customer accounts to place orders</li>
                 <li>Admin - Added product-details popover to allow quick visibility into products bought in an order without opening the order</li>
                 <li>Admin - Validation for integer min and max values and email addresses</li>
                 <li>Admin - Add sort order to order statuses so the admin order update dropdown can mirror your business workflow</li>
                 <li>Admin - Improve many configuration setting descriptions</li>
                 <li>Admin - Help buttons now link to new documentation</li>
+                <li>Admin – Products to Categories Link Manager. Addition of global tool to copy linked categories to another product.</li>
+                <li>Admin – Product Copy. Expanded to include copy of metatags and linked categories.</li>
+                <li>Admin - removing need for a separate set of language-specific email templates.</li>
```

Comment:
? Can you elaborate?

Reply:
@torvista Your feature - can you provide some more details?

Reply:
"Dear"
was hard-coded in several templates, thus requiring a pack of /es templates.
Not anymore.

Reply:
Updated language to be more specific on this point.